bot/resources/tags/dict-get.md

```diff
@@ -0,0 +1,15 @@
+Often while using dictionaries in Python, you may run into `KeyErrors`. This error is raised when you try to access a key that isn't present in your dictionary.\
+While you can use a `try` and `except` block to catch the `KeyError`, Python also gives you some other neat ways to handle them.
+
+__**The `dict.get` method**__
```

Comment:
```suggestion
**Providing default values**
```
We don't really underline sections in the tags. They're generally just bolded, so this would seem more consistent.

The change of wording isn't necessary, but it's something to consider.